ddl/db_integration_test.go

```diff
@@ -416,6 +416,14 @@ func (s *testIntegrationSuite5) TestMySQLErrorCode(c *C) {
 	assertErrorCode(c, tk, sql, tmysql.ErrPrimaryCantHaveNull)
 	sql = "create table t2 (id int auto_increment);"
 	assertErrorCode(c, tk, sql, tmysql.ErrWrongAutoKey)
+	sql = "create table t2 (a datetime(2) default current_timestamp(3))"
```

Comment:
Please check the following statements and add related test cases:

```
mysql> show columns from t;
+-------+-------------+------+-----+----------------------+--------------------------------+
| Field | Type        | Null | Key | Default              | Extra                          |
+-------+-------------+------+-----+----------------------+--------------------------------+
| a     | datetime    | NO   |     | CURRENT_TIMESTAMP    | on update CURRENT_TIMESTAMP    |
| b     | datetime(3) | NO   |     | CURRENT_TIMESTAMP(3) | on update CURRENT_TIMESTAMP(3) |
+-------+-------------+------+-----+----------------------+--------------------------------+
2 rows in set (0.01 sec)

mysql> select * from information_schema.columns where table_name = 't' and column_name='b'\G
*************************** 1. row ***************************
           TABLE_CATALOG: def
            TABLE_SCHEMA: test
              TABLE_NAME: t
             COLUMN_NAME: b
        ORDINAL_POSITION: 2
          COLUMN_DEFAULT: CURRENT_TIMESTAMP(3)
             IS_NULLABLE: NO
               DATA_TYPE: datetime
CHARACTER_MAXIMUM_LENGTH: NULL
  CHARACTER_OCTET_LENGTH: NULL
       NUMERIC_PRECISION: NULL
           NUMERIC_SCALE: NULL
      DATETIME_PRECISION: 3
      CHARACTER_SET_NAME: NULL
          COLLATION_NAME: NULL
             COLUMN_TYPE: datetime(3)
              COLUMN_KEY: 
                   EXTRA: on update CURRENT_TIMESTAMP(3)
              PRIVILEGES: select,insert,update,references
          COLUMN_COMMENT: 
   GENERATION_EXPRESSION: 
1 row in set (0.01 sec)

```

Reply:
Updated. PTAL @bb7133